Dockerfile.dev

```diff
@@ -5,10 +5,12 @@ RUN apt-get install -y git
 RUN apt-get install -y wget curl
 RUN apt-get install -y python3-pip
 
+ENV PIP_INDEX_URL="http://pypi.douban.com/simple/" PIP_TRUSTED_HOST="pypi.douban.com"
```

Comment:
We probably shouldn't modify pip index URL in the Dockerfile.

Reply:
agree